### DISCUSSION TITLE
FastApi Python swagger always returns blank page starlette 0.27.0

### DISCUSSION THREAD

**Discussion by BraintoByte** [fastapi/fastapi#10752] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import os
    from typing import Annotated
    from fastapi.responses import JSONResponse
    from fastapi import FastAPI
    from typing import Union
    
    
    app = FastAPI()
    
    @app.get("/")
    def read_root():
        return {"Hello": "World"}
    
    @app.get("/something")
    async def something():
        
        return {"message": "Hello World"}
```

### Description

I already checked, this is not a duplicate, this is happening with ```starlette 0.27.0```

Swagger is giving me a blank page when running. Everything looks fine on the bakcend

```
    INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
    INFO:     Started reloader process [24112] using WatchFiles
    INFO:     Started server process [14948]
    INFO:     Waiting for application startup.
    INFO:     Application startup complete.
    INFO:     127.0.0.1:36447 - "GET /redoc HTTP/1.1" 200 OK
    INFO:     127.0.0.1:36447 - "GET /docs HTTP/1.1" 200 OK
    INFO:     127.0.0.1:36447 - "GET /docs HTTP/1.1" 200 OK
```

This happens with docker and without it

```Python
    import os
    from typing import Annotated
    from fastapi.responses import JSONResponse
    from fastapi import FastAPI
    from typing import Union
    
    
    app = FastAPI()
    
    @app.get("/")
    def read_root():
        return {"Hello": "World"}
    
    @app.get("/something")
    async def something():
        
        return {"message": "Hello World"}```
```

Running:

```
Python 3.10 and 3.12
fastapi==0.104.1
uvicorn==0.24.0.post1
starlette==0.27.0
websockets==12.0
```

Error in console when going to /docs is:

[![Error in console Swagger FastApi][1]][1]

I will post this on github as well to let the devs of Swagger know, just wanted to do a sanity check

Any ideas?

  [1]: https://i.stack.imgur.com/M9qIh.png

### Operating System

Windows

### Operating System Details

Linux and windows

### FastAPI Version

0.104.1

### Pydantic Version

2.5.2

### Python Version

3.10 and 3.12

**Comment by jvdavim** [comment, marked as answer]
I'm using MacOS and couldn't reproduce. Maybe it's something related to your environment. It seems you are not able to access the CDN files to generate the Swagger UI. Check your connection to the internet or download the static swagger-ui-bundle files as showed on the FastAPI documentation: https://fastapi.tiangolo.com/how-to/custom-docs-ui-assets/.
![image](https://github.com/tiangolo/fastapi/assets/16657663/8c3477e7-b3eb-4903-9a75-5c4369815e7f)
